src/zenml/zen_server/deploy/terraform/recipes/gcp/variables.tf

```diff
@@ -125,10 +125,13 @@ variable "database_ssl_verify_server_cert" {
   type        = bool
 }
 
-variable "ingress_path" {
-  description = "The path on the Ingress URL to expose ZenML at"
-  default     = "zenml"
-  type        = string
+
+# Enable secrets manager API. Listing services might need elevated permissions.
+# Disable this if you don't have the ListServices permission.
```

Comment:
Given that we should be deprecating ZenML secrets managers anyway, is this even necessary for the secret store?

Reply:
This API is used for the new secret store in the ZenServer and not for the deprecated secrets manager. Disabling this would mean that the project that the user is deploying to, should already have this API enabled (probably by someone who has those permissions). 
Keeping it optional is important because not everybody running this recipe has elevated permissions.